sqlglot/dialects/bigquery.py

```diff
@@ -40,7 +40,7 @@ def func(self, expression):
 
 
 def _derived_table_values_to_unnest(self: generator.Generator, expression: exp.Values) -> str:
-    if not isinstance(expression.unnest().parent, exp.From):
+    if not isinstance(expression.parent, exp.From):
```

Comment:
This is a no-op because we know `expression` is an instance of `Values` here, so went ahead and fixed it while I was at it.

Reply:
just because expression is a values, doesn't mean it's parent is a from? i'm not sure i understand

Reply:
I mean that `expression.unnest()` seems like a no-op here: `expression` is of type `exp.Values` which doesn't override `unnest`.

Reply:
Makes sense, will update